README.md

```diff
@@ -94,12 +51,68 @@ This will inject `APP_KEY` and `DATABASE_URL` using your service name and stage,
 
 Finally, deploy your app:
 
+```bash
+serverless deploy
 ```
-sls deploy --stage=staging
+
+You can deploy to different environments (aka "stages") by using the `--stage` option:
+
+```bash
+serverless deploy --stage=staging
 ```
 
 Check out some more [comprehensive examples](examples/).
 
+## Octane
```

Comment:
Moved Octane and Queues to follow-up sections. Users also don't have to install the runtime by default anymore, which is one less step in the install.

Reply:
Almost everyone is going to want to use queues, I think. But fair enough.

Reply:
FYI I think I have a solution so that these two work out of the box (without setting up the runtime), but at this point I wanted to keep it out of this big PR

Reply:
@georgeboot here is what I meant: #43

I have applied the idea to the Octane runtime, but after that I'll apply it to the SQS runtime too. For Laravel users it will be a matter of putting the service in the `handler` field (they could put _any_ service btw).